docker-compose.yml

```diff
@@ -1,22 +1,86 @@
 version: '3.2'
 services:
   geth:
-    image: "matterlabs/geth:latest"
+    image: "ethereum/client-go:v1.13.5"
     ports:
-      - "8545:8545"
-      - "8546:8546"
+      - 8551:8551
+      - 8545:8545
+      - 8546:8546
     volumes:
-    - type: bind
-      source: ./volumes/geth
-      target: /var/lib/geth/data
+      - ./volumes/geth:/var/lib/geth/data
+      - ./docker/geth/:/geth/:ro
+    command:
+      - --networkid=9
+      - --datadir=/var/lib/geth/data
+      - --http
+      - --http.api=engine,eth,web3,personal,net,debug
+      - --http.addr=0.0.0.0
+      - --http.corsdomain=*
+      - --http.vhosts=*
+      - --ws
+      - --ws.addr=0.0.0.0
+      - --ws.port=8546
+      - --ws.origins=*
+      - --nodiscover
+      - --authrpc.addr=0.0.0.0
+      - --authrpc.vhosts=*
+      - --authrpc.jwtsecret=/var/lib/geth/data/jwtsecret
+      - --allow-insecure-unlock
+      - --unlock=0x8a91dc2d28b689474298d91899f0c1baf62cb85b
+      - --password=/var/lib/geth/data/password.sec
+      - --syncmode=full
+    depends_on:
+      beacon-chain:
+        condition: service_started
+  beacon-chain:
+    image: "gcr.io/prylabs-dev/prysm/beacon-chain:v4.1.1"
+    command:
+      - --datadir=/consensus/beacon/
+      - --min-sync-peers=0
+      - --genesis-state=/consensus/genesis.ssz
```

Comment:
This flag will actually allow the node to run only on the first launch. On subsequent relaunches it has to be commented out, I haven't yet found a good solution for this.

Reply:
No restarts will ever work anyway, so this is no longer relevant.